pkg/manager/manager.go

```diff
@@ -237,8 +240,26 @@ func (m MediaManager) AddMovieToLibrary(ctx context.Context, request AddMovieReq
 
 	slices.SortFunc(releases, sortReleaseFunc())
 	chosenRelease := releases[len(releases)-1]
+
 	log.Info("found release", "title", chosenRelease.Title)
-	return chosenRelease, nil
+
+	downloadClient, err := m.GetDownloadClient(ctx, int64(request.DownloadClientID)) // yolo cast?
```

Comment:
Which download client to use is going to depend on the chosen release right? Like torrent vs usenet. Can be a future problem

Reply:
I did add a type on the schema so we'll be able to query on something like torrent vs usenet, just wasn't sure how this wa going to look